src/app/achievements/achievements.component.ts

```diff
@@ -36,11 +36,11 @@ export class AchievementsComponent extends RouterEnter {
       image: '',
       name: ''
     };
+    this.getUser();
```

Comment:
Don't need to create a function here, just assign the correct value to `this.userInfo` above